kratos/python/add_constraint_to_python.cpp

```diff
@@ -7,7 +7,7 @@
 //  License:		 BSD License
 //					 Kratos default license: kratos/license.txt
 //
-//  Main authors:    Aditya Ghantasala
+//  Main authors:    Riccardo Rossi
```

Comment:
Why removing Aditya?

Reply:
I created this file for new in #6099 Aditya is there only because i copied a file in the same directory which happend to have him as author and forgot changing.